image/zero_crop_and_center/zero_crop_and_center.py

```diff
@@ -73,11 +73,11 @@ def center_in_image(image_array, output_size):
                          f'Got {len(output_array.shape)} array.')
 
     if output_array.shape[0] <= image_array.shape[0]:
-        raise ValueError(f'Input array must not be taller than input array.\n'
+        raise ValueError(f'Ouput array must not be taller than input array.\n'
```

Comment:
Thanks for the fix. This should be "Output" though, not "Ouput" :)

Reply:
Thanks for pointing out. I fixed it. Also, there's an "f" prefix that was omitted below.